cmake/Hunter/config.cmake

```diff
@@ -38,3 +38,14 @@ hunter_config(
     CMAKE_ARGS TESTING=OFF EXAMPLES=OFF EXPOSE_MOCKS=ON
     KEEP_PACKAGE_SOURCES
 )
+
+hunter_config(
+    GTest
+    URL https://github.com/google/googletest/archive/refs/tags/release-1.11.0.tar.gz
+    SHA1 7b100bb68db8df1060e178c495f3cbe941c9b058
+    CMAKE_ARGS
+      HUNTER_INSTALL_LICENSE_FILES=LICENSE
+      CMAKE_CXX_FLAGS=-Wno-unknown-warning-option
+      gtest_force_shared_crt=TRUE
+    KEEP_PACKAGE_SOURCES
```

Comment:
Do we really need the Gtest package sources? It was ok for libp2p and soralog, but why we need them for gtest?